.mailmap

```diff
@@ -1,6 +1,7 @@
 Adam Suban-Loewen <adam.subanloewen@gmail.com>
 
 Alexander Holas <alexander.holas@h-its.org>
+Alexander Holas <alex.holas@gmx.de>
```

Comment:
I think should be:

`Alexander Holas <alexander.holas@h-its.org> Alexander Holas <alex.holas@gmx.de>`